cmd/antrea-controller/controller.go

```diff
@@ -142,15 +158,79 @@ func run(o *Options) error {
 		networkPolicyStatusController = networkpolicy.NewStatusController(crdClient, networkPolicyStore, cnpInformer, anpInformer)
 	}
 
+	var anpMirroringController *crdmirroring.Controller
```

Comment:
Probably better to move these lines and line 129 to 140 to a separate func.

Reply:
Thanks for advices.  This could be better.